catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -14,16 +14,19 @@
 -->
 <metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
 
-    <OCD name="CSW Federated Source" id="Csw_Federated_Source" description="CSW Federated Source">
+    <OCD name="CSW Specification Profile Federated Source" id="Csw_Federated_Source"
```

Comment:
What does the "Specification Profile" portion of this new naming convention mean, or how does this indicate that it no longer includes the Connection Source stuff?